packages/BUILD.bazel

```diff
@@ -748,6 +748,12 @@ latex_package(
     tests = ["longtable_test.tex"],
 )
 
+latex_package(
+    name = "lstautogobble",
+    srcs = ["@texlive_texmf__texmf-dist__tex__latex__lstaddons"],
```

Comment:
add ":listings" to srcs and rebase on latest master. that did, at least locally, work for me.

Reply:
Interesting, I'm not sure I understand why that is needed but I've added it now, thanks!